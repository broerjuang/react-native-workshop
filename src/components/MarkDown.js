// @flow

import React from 'react';
import {StyleSheet} from 'react-native';
import Markdown, {styles} from 'react-native-markdown-renderer';
type Props = {
  content: string;
};
function markDown(props: Props) {
  let {content} = props;
  return <Markdown style={styles}>{content}</Markdown>;
}

export default markDown;

const syles = StyleSheet.create({
  heading: {
    borderBottomWidth: 1,
    borderColor: '#000000',
  },
  heading1: {
    fontSize: 32,
    backgroundColor: '#000000',
    color: '#FFFFFF',
  },
  heading2: {
    fontSize: 24,
  },
  heading3: {
    fontSize: 18,
  },
  heading4: {
    fontSize: 16,
  },
  heading5: {
    fontSize: 13,
  },
  heading6: {
    fontSize: 11,
  },
});
