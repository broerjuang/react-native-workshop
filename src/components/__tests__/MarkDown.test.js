import React from 'react';
import renderer from 'react-test-renderer';
import MarkDown from '../MarkDown';

describe('Icon', () => {
  it('should render EventCard corectly', () => {
    let component = renderer.create(<MarkDown content={'hello '} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
