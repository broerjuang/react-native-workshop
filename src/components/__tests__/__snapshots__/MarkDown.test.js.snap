// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render EventCard corectly 1`] = `
<View
  style={Object {}}
>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "flex-start",
        "marginBottom": 10,
        "marginTop": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={Object {}}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        hello
      </Text>
    </Text>
  </View>
</View>
`;
